hydra_openapi_parser/openapi_parser.py

```diff
@@ -103,7 +107,7 @@ def valid_endpoint(path: str) -> str:
     :return:
     """
     # "collection" or true means valid
-    path_ = path.split('/')
+    path_ = path.split("/")
```

Comment:
you should be sure that the rules for the formatter stick to the conventions we established for the codebase. For example: we decide to always use `'` single quotes for strings, except that for dictionaries meant to be dumped into JSON strings and for variables added via f-strings like `f'Some text with a {dict["variable"]}'`

Reply:
https://www.python.org/dev/peps/pep-0008/#string-quotes